pkg/services/rendering/http_mode.go

```diff
@@ -155,6 +155,9 @@ func (rs *RenderingService) doRequest(ctx context.Context, url *url.URL, headers
 	resp, err := netClient.Do(req)
 	if err != nil {
 		rs.log.Error("Failed to send request to remote rendering service", "error", err)
+		if os.IsTimeout(err) {
```

Comment:
If I understood it correctly, [the Go documentation](https://pkg.go.dev/os#IsTimeout) says this function should no longer be called but use the `errors.Is` instead. Does that make sense? 🤔

Reply:
In `Do()` function documentation says:
```
// Any returned error will be of type *url.Error. The url.Error
// value's Timeout method will report true if the request timed out.
```

So I mapped it to `*url.Error` to check the error.

Reply:
Great, thanks for the explanation, good to know! 😄